logger/logger.go

```diff
@@ -0,0 +1,21 @@
+package logger
```

Comment:
I'm not sure this is already worth extracting into its own package, I think just having it inside `logger.go` in main package would be fine

Reply:
It's just a short optimization to future refactoring